CourseGrab/AppDelegate.swift

```diff
@@ -6,6 +6,8 @@
 //  Copyright © 2020 Cornell AppDev. All rights reserved.
 //
 
+import Firebase
+import FirebaseAuth
```

Comment:
ALPHABETIZE 🤪

Reply:
@ReadeP. I don't know how to express this. However, I will try.

<details>
<summary>This might help.</summary>

![3pvr6j](https://user-images.githubusercontent.com/23707637/75046104-8023a700-5492-11ea-82dc-ad5ada4e5a0c.jpg)

</details>

Reply:
<details>
<summary>Oh wow.</summary>

![download](https://user-images.githubusercontent.com/19507508/75056651-78213280-54a5-11ea-89a9-d68cb0c929eb.jpg)


</details>